packages/apidom-ls/test/openapi-json.ts

```diff
@@ -475,6 +484,40 @@ describe('apidom-ls', function () {
         severity: 1,
         source: 'apilint',
       },
+      // {
```

Comment:
Can you please remove this commented structure? It has been replaced by other up-to-date structures.